src/navigation.html

```diff
@@ -194,317 +194,353 @@ <h3 id="sub-nav-vertical">Vertical</h3>
   <p>Vertical sub-navs should be a portion of the page width, and appear next to the content in which they navigate. If an item is disabled or unavailable, but must still be displayed, be sure to provide a visual cue that the element is not clickable, as well as contextually-appropriate hover effects.</p>
 
   <div class="bs-example" data-example-id="sub-nav-vert">
-    <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
-      <nav>
-        <ul class="nav es-nav sub-nav vertical">
-          <li role="presentation" class="active-example"><a href="javascript:;">Family Members <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation"><a href="javascript:;">Applications &amp; Policies <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation"><a href="javascript:;">Funds &amp; Reimbursements <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation" class="disabled"><a href="javascript:;">Account Settings <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-        </ul>
-      </nav>
-    </div>
-    <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
-      <nav>
-        <ul class="nav es-nav sub-nav vertical light">
-          <li role="presentation" class="active-example"><a href="javascript:;">Family Members <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation"><a href="javascript:;">Applications &amp; Policies <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation"><a href="javascript:;">Funds &amp; Reimbursements <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-          <li role="presentation" class="disabled"><a href="javascript:;">Account Settings <span class="icon chevron-right" aria-hidden="true"></span></a></li>
-        </ul>
-      </nav>
+    <div class="row">
```

Comment:
@b-kerr – there are some new classes in here that leverage BS4 to fix display issues. Not sure how these manifested, seemingly randomly. Let's discuss on Friday.